README.md

```diff
@@ -1 +1,35 @@
-# Hello-world
\ No newline at end of file
+Hello Microverse
+The project is about Microverse courses.
+
+Built With
+Javascript, CSS, HTML
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+👤 Author1
+
+GitHub: @Victoria129
+Twitter: @Tumwebaze129
+LinkedIn: victoria-tumwebaze-01701424a
```

Comment:
Kindly note that the above should link to your github page, twitter handle and your linkedin profile since you have one

To acheieve this use the following syntax:
   `Github: [@Victoria129](https://github.com/Victoria129)`
    `Twitter:[@Tumwebaze129](https://twitter.com/Tumwebaze129)`
   ` Linkedin:[Linkedi](linkein link)`

Your username should be a link to open your social media pages. A potential recruiter may want to get in touch with you and having this as only usernames means you might just miss out on an opportunity.